src/vsphere_cpi/spec/integration/datastore_migration_spec.rb

```diff
@@ -136,8 +136,7 @@
         mismatch_cpi.create_vm('agent-007', @stemcell_id, vm_type, network_spec)
       }.to raise_error { |error|
         expect(error).to be_a(Bosh::Clouds::CloudError)
-        expect(error.message).to include(@second_cluster_datastore)
-        expect(error.message).to include('No valid placement found for disks')
+        expect(error.message).to include('No valid placement found')
```

Comment:
Let's take this opportunity to fix this code to:

```ruby
}.to raise_error(Bosh::Cloud::CloudError, /No valid placement found/)
```

Reply:
Fixed